python/pyarrow/_csv.pyx

```diff
@@ -57,7 +57,7 @@ cdef class ReadOptions(_Weakrefable):
     block_size : int, optional
         How much bytes to process at a time from the input stream.
         This will determine multi-threading granularity as well as
-        the size of individual chunks in the Table.
+        the size of individual record batches.
```

Comment:
For the non-streaming reader, this is still "chunks in the Table" ?

Reply:
Right, but those are similar to record batches in this case.